Sources/Controllers/TargetMenu/Search/OAQuickSearchCoordinatesViewController.mm

```diff
@@ -134,6 +131,26 @@ - (instancetype) initWithLat:(double)lat lon:(double)lon
 - (void)viewWillAppear:(BOOL)animated
 {
     [super viewWillAppear:animated];
+    
+    [self.navigationController setNavigationBarHidden:NO animated:YES];
```

Comment:
The code for styling the navigation bar should be placed in a separate method, configureNavigationBar